### DISCUSSION TITLE
Unable to sort or filter PRs via author in new Projects boards queries

### DISCUSSION THREAD

**Discussion by dtzar** [community/community#41038] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

If I am on the `/pulls` page OR the global GitHub search pages, I am easily able to filter utilizing `author:<name>`.  When I do this with PRs which have been added to the new projects boards, this does not work at all.  Filtering via reviewers or assignees does not suffice.  I furthermore cannot add/list a column of the author of the PR in a separate column.

**Comment by ossanna16** [comment]
Hi there @dtzar and welcome to our community! Thank you for the feedback!

**Comment by ajtowns** [comment]
This...
